src/gui.rs

```diff
@@ -165,25 +153,46 @@ impl Application for State {
             Command::batch([
                 Command::perform(fetch_minecraft_versions(), Message::SetMcVersions),
                 Command::perform(fetch_loader_versions(), Message::SetLoaderVersions),
-            ])
+            ]),
         )
     }
 
     fn title(&self) -> String {
-        "Quilt Installer".to_owned()
+        format!(
+            "Quilt Installer{}",
+            if self.is_installing {
+                " - Installing"
+            } else {
+                ""
+            }
+        )
```

Comment:
Hm, I'd rather have a consistent window name here, with the possible use of OS trickery in order to make the app icon show the progress (like that green thing on Windows)